packages/workspace/src/utils/filenameutils.ts

```diff
@@ -0,0 +1,7 @@
+import { parse } from 'path';
```

Comment:
Please move this to a deep import from `@nrwl/devkit` instead of `@nrwl/workspace`.

Reply:
Done. Thanks for the advice.
Also, what's wrong with `main-linux` pipeline? I can't have it succeed.